app/models/reports/export_raw.rb

```diff
@@ -33,55 +33,55 @@ def default_report_hash
       hash = {
         facility: :facility,
         order: :to_s,
-        ordered_at: -> (od) { od.order.ordered_at },
-        fulfilled_at: -> (od) { od.fulfilled_at },
-        order_status: -> (od) { od.order_status.name },
+        ordered_at: ->(od) { od.order.ordered_at },
```

Comment:
I've been avoiding rubocopping this because of the annoyance of merge conflicts as I've been touching this quite a bit for rooms, but I think it's a worthwhile change. So we'll see who gets this in first!